.eslintrc.json

```diff
@@ -2,14 +2,41 @@
   "extends": [
     "next/core-web-vitals",
     "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
```

Comment:
Are these two plugins necessary to specify here (react/recommended and react-hooks/recommended)? Looks like they should already be included via next's eslint config which we're extending. https://nextjs.org/docs/pages/building-your-application/configuring/eslint#eslint-config